src/cljs/athens/devcards/db_boxes.cljs

```diff
@@ -301,27 +151,27 @@
            :style {:cursor :pointer}}
        (str value)]
 
-      (attr-unique? attr)
+      (brws/attr-unique? attr)
       [:a {:on-click #(update-and-eval-box! (pull-entity-str attr value))
```

Comment:
Looks like we can't nav with `edn` view. Is that something we want to support?

Reply:
We could of course, but it won't be easy to implement. Right now the output of the `edn` view is just a `string`. To make it navigable we would have to implement something to convert the data structure into hiccup with the right clickable parts.

I would suggest postponing this to a later stage if we really want it.